src/ConfigWidgets/SymbolErrorDialog.cpp

```diff
@@ -19,9 +19,18 @@ SymbolErrorDialog::SymbolErrorDialog(const orbit_client_data::ModuleData* module
     : QDialog(parent), ui_(std::make_unique<Ui::SymbolErrorDialog>()), module_(module) {
   ORBIT_CHECK(module_ != nullptr);
   ui_->setupUi(this);
-  ui_->moduleNameLabel->setText(QString::fromStdString(module_->file_path()));
+  QString module_file_path = QString::fromStdString(module_->file_path());
+  ui_->moduleNameLabel->setText(module_file_path);
   ui_->errorPlainTextEdit->setPlainText(QString::fromStdString(detailed_error));
-  ui_->addSymbolLocationButton->setFocus();
+  if (!module_->build_id().empty()) {
+    ui_->addSymbolLocationButton->setFocus();
+  } else {
```

Comment:
Is it expected behavior that there is not `setFocus()` call in this block? I get that the element you're focussing in the code above is disabled, so do we rely on the default focus set in QT Designer, or do we just not care? Why do we care in the other case?

Reply:
For reasons I really do not understand (and I tried to dig for way too long), Qt does not want to focus that button simply based on the UI file. In the `SymbolsDialog` I did exactly the same thing and it works flawlessly.. So yeah, I decided to do the slightly uncool focus setting by hand here. In case you have an idea what it could be. I am all ear.

Regarding the functionality: If there is a build-id (`!.empty`), then the focus is set. When there is no build-id, the button is disabled and we dont care about the focus.